lib/ansible/plugins/lookup/dsv.py

```diff
@@ -0,0 +1,128 @@
+# python 3 headers, required if submitting to Ansible
```

Comment:
@amigus Could you please submit this PR [here](https://github.com/ansible-collections/community.general)?

Reply:
Done. Sorry for my mistake.  Thanks!